plugins/org.yakindu.sct.model.stext/src/org/yakindu/sct/model/stext/validation/STextValidationMessages.java

```diff
@@ -54,8 +54,9 @@
 	public static final String INTERNAL_DECLARATION_UNUSED = "Internal declaration is not used in statechart.";
 	public static final String ERROR_WRONG_ANNOTATION_TARGET_CODE = "WrongAnnotationTarget";
 	public static final String ERROR_WRONG_ANNOTATION_TARGET_MSG = "Annotation '%s' can not be applied on %s ";
-	
 	public static final String VALUE_OF_REQUIRES_EVENT = "valueof() expression requires event as argument.";
+	public static final String VAR_ARGS_LAST_CODE = "VarArgsMustBeLast";
+	public static final String VAR_ARGS_LAST_MSG = "The variable argument type must be the last argument.";
```

Comment:
Error message could be clearer, i.e. something like:
"Variadic function argument must be the last argument."